### PR TITLE
[NTP] Configure NTP according to extended configuration

### PR DESCRIPTION
**Pull request by fastiuk**
hld [#1219](https://github.com/sonic-net/SONiC/pull/1296)
closes [#1171](https://github.com/sonic-net/SONiC/issues/1254)

#### What I did
Fix NTP CLI according to new extended configuration abilities

#### How I did it
Additionally configure `association_type` and `resolve_as` when adding new NTP servers.

#### How to verify it
* Add a new server
* Show NTP sync `show ntp`. It should show you are sync with new server
* Delete NTP server
* Show NTP sync `show ntp`. It should show you are unsync

#### New command output (if the output of a command-line utility has changed)
Not changed

### CODE REVIEW

File: config/main.py [hunk 1]
```diff
@@ -6515,7 +6515,9 @@ def add_ntp_server(ctx, ntp_ip_address):
         return
     else:
         try:
-            db.set_entry('NTP_SERVER', ntp_ip_address, {'NULL': 'NULL'})
+            db.set_entry('NTP_SERVER', ntp_ip_address,
+                         {'resolve_as': ntp_ip_address,
+                          'association_type': 'server'})
```
Comment (saiarcot895) [inline config/main.py:6520]: Just to verify, this would mean that pool NTP servers can be configured only via `config_db.json` directly, right?

Comment (fastiuk) [reply config/main.py:6530]: Yes, right

**Review by saiarcot895 — APPROVED** [review]

**Comment by bsun-sudo** [comment]
Just want to make sure. Should this PR support all new configurations from the [HLD](https://github.com/sonic-net/SONiC/pull/1296), or there will be a separate PR?

**Comment by fastiuk** [comment]
> Just want to make sure. Should this PR support all new configurations from the [HLD](https://github.com/sonic-net/SONiC/pull/1296), or there will be a separate PR?

No, the new commands won't be supported by this PR

**Comment by liat-grozovik** [comment]
could it be that the NTP command now looks different?
if so, https://github.com/sonic-net/sonic-utilities/blob/master/doc/Command-Reference.md#ntp-config-commands should be updated and aligned with the extra options

**Review by liat-grozovik — APPROVED** [review]

**Pull request merged**